papers/michael_lange/michael_lange.rst

```diff
@@ -460,10 +488,17 @@ efficiently define rigorous forward modelling and adjoint operators
 from high-level symbolic definitions also implies that domain
 scientists are able to quickly adjust the numerical method and
 discretisation to the individual problem and hardware architecture
-[Louboutin17a]_. In the following example we will demonstrate the
-generation of forward and adjoint operators for the acoustic wave
-equation to implement the so-called adjoint test. The governing
-equation is defined as
+[Louboutin17a]_.
+
+In the following example we will demonstrate the generation of forward
+and adjoint operators for the acoustic wave equation and verify their
```

Comment:
Not entirely sure how you want to split the sentence. Can you please be more specific? Happy to add the Virieux reference though.